Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM python:latest
+FROM python:3.11
 
 WORKDIR /usr/src/app
 
-COPY src /usr/src/app/src
-COPY Makefile requirements.txt requirements-tests.txt setup.cfg /usr/src/app/
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
 
-RUN make venv
-
-CMD ["make", "run"]
\ No newline at end of file
+COPY src/ /usr/src/app/
+ENTRYPOINT [ "python", "./main.py" ]
```

Comment:
I don't see the point of no having a centralize file with all the available "commands" you can do

Reply:
I prefer to have the application on a docker image without any other files than the ones needed to run the application. I've created a separate Dockerfile to run tests.

Reply:
but then things like all the linter tools are not centralized in one file, honestly I don't see it